File: dataset/output-bk/CLAUDE_claude-3-5-sonnet-20240620/Task32_CLAUDE_claude-3-5-sonnet-20240620.java

public class YearsToDesiredSum {
    public static int calculateYears(double principal, double interest, double tax, double desired) {
        int years = 0;
        while (principal < desired) {
            double interestEarned = principal * interest;
            double taxPaid = interestEarned * tax;
            principal += interestEarned - taxPaid;
            years++;
        }
        return years;
    }

    public static void main(String[] args) {
        System.out.println(calculateYears(1000, 0.05, 0.18, 1100));
    }
}

File: dataset/output/CLAUDE_claude-3-5-sonnet-20240620/Task32_CLAUDE_claude-3-5-sonnet-20240620.c

int calculate_years(double principal, double interest, double tax, double desired) {
    int years = 0;
    while (principal < desired) {
        double interest_earned = principal * interest;
        double tax_paid = interest_earned * tax;
        principal += interest_earned - tax_paid;
        years++;
    }
    return years;
}
